app/javascript/components/patient/vaccines/VaccineModal.js

```diff
@@ -224,6 +261,18 @@ class VaccineModal extends React.Component {
   }
 }
 
+const schema = yup.object().shape({
+  group_name: yup.string().required('Please select a vaccine group.').max(200, 'Max length exceeded, please limit to 200 characters.').nullable(),
+  product_name: yup.string().required('Please select a product name.').max(200, 'Max length exceeded, please limit to 200 characters.').nullable(),
+  administration_date: yup
+    .date('Date must correspond to the "mm/dd/yyyy" format.')
+    .min(moment('1900-01-01'), 'Specimen Collection Date must fall after January 1, 2020.')
```

Comment:
2020-01-01 instead of `'1900-01-01'` ?

Reply:
`1900-01-01` is the `minDate` on the datepicker: https://github.com/SaraAlert/SaraAlert/blob/1527-labs-table-refactor/app/javascript/components/patient/vaccines/VaccineModal.js#L178

@tstrass @hackrm let me know if you think this should be changed

Reply:
Since the Data Dictionary mentions the valid date range for this field goes to `1900-01-01`, so you can leave it as is.

![image](https://user-images.githubusercontent.com/42656458/149366856-64c55acc-77f4-4c7a-ab1f-b0f486d02316.png)